Source/lighting.cpp

```diff
@@ -351,12 +351,12 @@ void MakeLightTable()
 		const auto shades = static_cast<int>(LightTables.size() - 1);
 		for (int i = 0; i < shades; i++) {
 			auto &lightTable = LightTables[i];
-			constexpr int range = 16;
-			for (int j = 0; j < range; j++) {
-				uint8_t color = ((range - 1) << 4) / shades * (shades - i) / range * (j + 1);
+			constexpr int RANGE = 16;
+			for (int j = 0; j < RANGE; j++) {
+				uint8_t color = ((RANGE - 1) << 4) / shades * (shades - i) / RANGE * (j + 1);
 				color = 1 + (color >> 4);
-				lightTable[j + 1] = color;
-				lightTable[31 - j] = color;
+				lightTable[static_cast<uint8_t>(j + 1)] = color;

```

Comment:
couldn't you just make `j` a uint8_t to start with so you don't need to static cast it?

Reply:
I will try but I think that still might have a compiler warning

Reply:
![image](https://github.com/diasurgical/devilutionX/assets/68359262/dae26a1e-9140-46e6-bcd4-6400c1bfb23a)
Doesn't fix the issue